var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -27,28 +27,31 @@
 
 class Paraview(Package):
```

Comment:
Sorry, I did try that, but everything seemed to fall apart on me. I think that the cmake_args were getting stepped on somewhere. At least the reworked version is _really_ close to being a CMakePackage. But I'd really like to get this merged ASAP so that I can followup on my OpenFOAM packages (3 of them now), which depend on getting the includes from this updated paraview package.